ansible/roles/osdsdb/scenarios/etcd.yml

```diff
@@ -29,38 +29,39 @@
   get_url:
     url={{ etcd_download_url }}
     dest=/opt/{{ etcd_tarball }}
-  when:
```

Comment:
Any reason for changing these ?

Reply:
These got changed by mistake. Essentially the same thing but written in one line. I have reverted it in a commit.